include/mppic/critics/prefer_forward_critic.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+// Copyright 2022 FastSense, Samsung Research
```

Comment:
Copyright has to be the first line or linters will complain

Reply:
Done